fpga/lib/core.cpp

```diff
@@ -37,21 +37,10 @@ static std::list<Vlnv> vlnvInitializationOrder = {
     Vlnv("xilinx.com:ip:axi_iic:"),
 };
 
-std::list<std::shared_ptr<Core>> CoreFactory::make(Card *card,
-                                                   json_t *json_ips) {
-  // We only have this logger until we know the factory to build an IP with
-  auto loggerStatic = getStaticLogger();
-
-  std::list<IpIdentifier> allIps;     // All IPs available in config
-  std::list<IpIdentifier> orderedIps; // IPs ordered in initialization order
-
-  std::list<std::shared_ptr<Core>> configuredIps; // Successfully configured IPs
-
-  if (!card->ips.empty()) {
-    loggerStatic->error("IP list of card {} already contains IPs.", card->name);
-    throw RuntimeError("IP list of card {} already contains IPs.", card->name);
-  }
+std::list<IpIdentifier> CoreFactory::parseVLNV(json_t *json_ips) {
```

Comment:
Maybe better names would be `parseIpIdentifier` or 'getIpIdentifier'. Why not move this to a constructor of `IpIdentfier`? Then we can here only have a `getOrderedIpList` or something like that.

Reply:
I renamed the method to parseIpIdentifier. I dislike "get", since it doesnt get any member.
Core constructor should not parse anything, since it would break the single responsiblity principle.

Reply:
I meant the IpIdentifier Constructor. But maybe we don't want to bring the jansson dependency into that file.